qcfractal/interface/client.py

```diff
@@ -31,6 +45,19 @@ def __init__(self, address, username=None, password=None, shared_secret=None):
         else:
             self._auth = {}
 
+    def __repr__(self):
```

Comment:
Should be `__str__` instead? `__repr__` often is a string that can be used to re-instantiate the object.